pkgs/development/web/deno/default.nix

```diff
@@ -0,0 +1,59 @@
+{ stdenv, fetchurl, fetchFromGitHub, rustPlatform, python27 }:
+
+let
+  arch = if stdenv.isDarwin then "x86_64-apple-darwin" else "x86_64-unknown-linux-gnu";
```

Comment:
Can we add `aarch64-unknown-linux-gnu` here?

Reply:
Sure
I'm moving it over to a setup similar to firecracker ([link](https://github.com/NixOS/nixpkgs/blob/f83c5d3432d302125cd3f5ca2ee7f44934077fe0/pkgs/applications/virtualization/firecracker/default.nix)) because I noticed each of these libraries will have a different sha sum

Reply:
The changes have been pushed but I've not tested them on Mac or an aarch64 arm machine yet. I'll resolve these conversations & track the testing progress above in the `"Built on platform(s)" section